<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>异想家数据实验室</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>数据标准化</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>数据标准化</string>
      </property>
      <widget class="QPushButton" name="pushButton_DS18B20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>DS18B20(Dir)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_CCD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>CCD(Dir)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_FBGT">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>FBG温度(Dir)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_FBGS">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>FBG应力(Dir)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ENV">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>30</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>环境温度(*.xls)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>761</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>文件状态：未载入</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>20</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Dir:选文件夹
*.xls:多选xls</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>波长转温度(经供参考)</string>
      </property>
      <widget class="QPushButton" name="pushButton_LoadFBGT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>标准FBG文件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_wave2temp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>一键转换</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>未载入</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>相关性计算</string>
      </property>
      <widget class="QPushButton" name="pushButton_covresult">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>生成结果</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_File2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>文件2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_File1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string> 文件1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>待计算</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>数据概览</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>载入数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>清空全部</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>881</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>数据概览</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>661</width>
         <height>39</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>数据行列：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>数据总和：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>数据最大：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据名字：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>数据平均：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>数据最小：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>981</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>数据基本信息</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>961</width>
         <height>421</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>数据绘图</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>580</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>蒋方正制作</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
